github-dark.css

```diff
@@ -3715,6 +3715,9 @@
   .sidebar-notifications .thread-subscription-status {
     border: 0 !important;
   }
+  .select-menu-tabs ul .select-menu-tab-nav[aria-selected="true"] {
+    border-bottom: 1px solid transparent !important;
```

Comment:
Setting just the color with `border-bottom-color` works too which might be better than setting the full style.

Reply:
Done, and moved selector to existing rule with this value.